spec/integration/type/file_spec.rb

```diff
@@ -1528,4 +1528,161 @@ def pretty_transaction_error(transaction)
 
     raise "Got #{status_failures.length} failure(s) while applying: #{status_fail_msg}"
   end
+
+  describe "copying a file that is a link to a file" do
```

Comment:
This should be guarded to platforms that support `manages_symlinks` since 2003 does not (and newer versions don't if you're not running with privileges):

```
it "does not package with a symlink", :if => Puppet.features.manages_symlinks? do
```